fuzzing/gnrc_tcp/main.c

```diff
@@ -17,30 +17,44 @@
 #include "net/ipv6/addr.h"
 #include "net/gnrc/pkt.h"
 
+#define TCB_QUEUE_SIZE 1
+#define ACCEPT_TIMEOUT_MS 60000
+
 static uint32_t demux = GNRC_NETREG_DEMUX_CTX_ALL;
 static gnrc_nettype_t ntype = GNRC_NETTYPE_TCP;
 
 static void *tcploop(void *arg)
 {
     mutex_t *tcpmtx = arg;
-    gnrc_tcp_tcb_t tcb;
+    gnrc_tcp_tcb_queue_t queue = GNRC_TCP_TCB_QUEUE_INIT;
+    gnrc_tcp_tcb_t tcbs[TCB_QUEUE_SIZE];
```

Comment:
This could get very large when `TCB_QUEUE_SIZE` is increased and thus increase the stack usage. I don't see any reason why this array has to be allocated on the stack, so maybe make it at least `static` or move it to the global context.

Reply:
Although I don't think that TCB_QUEUE_SIZE will ever change, I moved it to global context. The reason for the stack allocation was an attept to keep the application the way it was.